features/step_definitions/profile_update_steps.rb

```diff
@@ -0,0 +1,57 @@
+Given(/^The course page with (.+)$/) do |course_page|
+	@course = FactoryGirl.create(:course, title: course_page)
```

Comment:
indentation is off here and in the rest of steps, It's a common idiom in ruby to indent with two spaces. Can you change the indentation to follow that convention please 😄